install_config/configuring_sdn.adoc

```diff
@@ -212,37 +212,41 @@ $ oc get netnamespace
 ----
 
 [[migrating-between-sdn-plugins-networkpolicy]]
-=== Migrating from the ovs-multitenant Plugin to ovs-networkpolicy
+=== Migrating from ovs-multitenant to ovs-networkpolicy
 
-Before migrating from *ovs-multitenant* to *ovs-networkpolicy*, you must ensure that every
-namespace has a unique `NetID`; that means that if you have previously
+Before migrating from the *ovs-multitenant* plugin to the *ovs-networkpolicy*
+plugin, ensure that every namespace has a unique `NetID`. This means that if you
+have previously
 xref:../admin_guide/managing_networking.adoc#joining-project-networks[joined projects
 together] or
 xref:../admin_guide/managing_networking.adoc#making-project-networks-global[made projects
-global], you will need to undo that before switching to the *ovs-networkpolicy* plugin, or
-NetworkPolicy objects may not function correctly.
+global], you will need to undo that before switching to the *ovs-networkpolicy* plugin,
+ or the NetworkPolicy objects may not function correctly.
```

Comment:
(you ended up with a space at the beginning of that line... not sure if that causes anything weird to happen in asciidoc)